scripts/README.md

```diff
@@ -74,7 +74,8 @@ It also generates the `build/parachain.wasm` and `build/parachain.genesis`
 
 ## Running Relay nodes
 
-You can run up to 9 relay nodes with this script (which runs the docker image)
+You can run up to 3 relay nodes with this script (which runs the docker image).
```

Comment:
```suggestion
You can run up to 3 relay chain validators with this script. We use the `purestake/moonbase-relay-testnet` docker image for validators. Currently this image is manually published from commit (TODO), but this will change in the future.
```

Probably worth documenting _which_ docker image.